src/reducers/fileTree.tsx

```diff
@@ -261,7 +262,8 @@ const findRelativeMessage = (
         position === RelativePathPosition.previous ? msgArray.length - 1 : 0;
 
       if (msgArray[msgIndex]) {
-        return { path: nextPath, uid: msgArray[msgIndex].uid };
+        const message = msgArray[msgIndex];
+        return { line: message.line, path: nextPath, uid: message.uid };
```

Comment:
This (and the next one) would benefit from a shared arrow function you can call, like:

```ts
return makeRelativeMessageInfo({ message, path });
```

```ts
const makeRelativeMessageInfo = ({ message, path }) => {
  return { line: message.line, path, uid: message.uid };
};
```